corehq/apps/sso/views/enterprise_admin.py

```diff
@@ -41,6 +43,10 @@ class EditIdentityProviderEnterpriseView(BaseEnterpriseAdminView, AsyncHandlerMi
         SsoTestUserAdminAsyncHandler,
     ]
 
+    @use_jquery_ui  # for datepicker
+    def dispatch(self, request, *args, **kwargs):
+        return super(EditIdentityProviderEnterpriseView, self).dispatch(request, *args, **kwargs)
```

Comment:
Nit: do not specify class names in the `super()` call, i.e. `super().dispatch(...`

Reply:
Thanks! Fixed in a613273eb7480f04873965b634f35de833bf410c